src/main/config/conversion/import/iso19139.ca.HNAP.FGP-to-iso19139.ca.HNAP.xsl

```diff
@@ -231,6 +234,74 @@
     </xsl:copy>
   </xsl:template>
 
+  <!-- Change http to https for licence URLs -->
```

Comment:
Please reference that this is to address this issue https://github.com/metadata101/iso19139.ca.HNAP/issues/304